ci-operator/step-registry/ovn/conf/vsphere-dualstack/ovn-conf-vsphere-dualstack-commands.sh

```diff
@@ -35,3 +35,12 @@ networking:
   - 172.30.0.0/16
   - fd65:172:16::/112
 EOF
+
+if [[ "${IP_FAMILIES}" == "DualStackIPv6Primary" ]]; then
```

Comment:
It's weird that you have `IP_FAMILIES` but it's always either `""` or `"DualStackIPv6Primary"`...

Maybe make it `PRIMARY_IP_FAMILY` and have it default to `IPv4`?